docs/content/standards/coin.mdx

```diff
@@ -1,50 +1,77 @@
+
+Fungible tokens created on Sui using the Coin or Currency standard are referred to as _coins_. 
+
+For fungible tokens created on Sui using the [Closed-Loop Token standard](/standards/closed-loop-token.mdx), the documentation uses the term _tokens_. In practice, the terms for both these objects are often interchangeable.
 
 :::
 
-The Coin standard is a technical standard used for smart contracts on Sui for creating coins on the Sui blockchain. See [Sui Tokenomics](/concepts/tokenomics.mdx) to learn more about the SUI native coin and its use on the Sui network.
+The Coin Standard is a technical standard used by Move smart contracts for creating currency on Sui. The `sui::coin` module provides the logic that defines the standard.
 
-See the [coin module](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/docs/sui/coin.md) documentation for all available options when creating a coin-type token on Sui.
+The standardization of currency creation on Sui means that wallets, exchanges, and other smart contracts can manage currency created on Sui the same as they manage SUI, without any additional processing logic.
 
-:::info
+See [Sui Tokenomics](/concepts/tokenomics.mdx) to learn more about the SUI native currency and its use on the network.
 
-The documentation refers to fungible tokens created on Sui using the Coin standard as _coins_. For fungible tokens created on Sui using the [Closed-Loop Token standard](/standards/closed-loop-token.mdx), the documentation uses the term _tokens_. In practice, the terms for both these objects are often interchangeable.
+## Fungible tokens
 
-:::
+The `Coin<T>` type represents open-loop fungible tokens (see `Token<T>` for closed-loop tokens). Coins are denominated by their type parameter, `T`, which is also associated with metadata (like name, symbol, decimal precision, and so on) that applies to all instances of `Coin<T>`. The `sui::coin` module exposes an interface over `Coin<T>` that treats it as fungible, meaning that a unit of `T` held in one instance of `Coin<T>` is interchangeable with any other unit of `T`, much like how traditional fiat currencies operate.
+
+## Coin creation
+
+When you create a coin using the `coin::create_currency` function, the publisher of the smart contract that creates the coin receives a `TreasuryCap` object. 
+
+### Supply states
+
+The registry supports three different supply management models:
+
+- **Fixed supply:** The total supply is permanently locked and cannot be changed.
+- **Uncontrolled supply:** `TreasuryCap` holder controls minting and burning.
+
+### Regulatory states
+
+Currencies can have different regulatory states:
+
+- **Regulated:** The currency has an associated `DenyCapV2` that can restrict addresses from using it.
+- **Unregulated:** The currency was created without any deny list capabilities.
+- **Unknown:** Regulatory status is undetermined, typically from legacy migrations.
 
 ## Treasury capability
```

Comment:
This is out of the scope of this change, but can we add a `::danger` element with something like:

> While `TreasuryCap` is freely transferable, never freeze or share the `TreasuryCap`. Doing so may allow malicious actors to call functions as the currency owner and the latter allows anyone to mint new coins and act as the owner of the currency. 

Very poor wording on my end, happy to chat to shape it better.